src/frontend/js/components/LtiConsumer/_styles.scss

```diff
@@ -1,9 +1,29 @@
-.lti-consumer {
-  animation: fadeIn 600ms cubic-bezier(0.33, 1, 0.68, 1) forwards;
-  background-color: #ffffff;
+.richie-react--lti-consumer {
+  &.aspect-ratio {
+    position: relative;
+    padding-bottom: 56.25%; // Aspect ratio 16/9
```

Comment:
isn't the aspect ratio supposed to be passed via the template context?

Reply:
Yes, 16/9 is default value. Check `inline_ratio_css_padding_bottom` to see how this style is overrided.

Reply:
But there is a default value also in the context so maybe this is useless 🤔

Reply:
You are right, `padding-bottom` rule is always overridden in the LTIConsumerPlugin context.